tests/common.go

```diff
@@ -4107,11 +4152,24 @@ func AddLabelToResource(spec interface{}, key string, val string) error {
 	return fmt.Errorf("spec is of unknown resource type")
 }
 
+// GetClusterConfigPath returns kubeconfig for given cluster
+func GetClusterConfigPath(clusterConfig string) (string, error) {
```

Comment:
This function does not actually return any errors.

Reply:
done